app/models/text_field_type.rb

```diff
@@ -25,14 +28,23 @@ def mapping_field_name
   end
 
   def text_present
-    errors.add(:text, "must be present") if @text.empty?
+    errors.add(:text, 'must be present') if @text.empty?
```

Comment:
@ElliottAYoung I didn't write this code, it was already there. The change must be from atom beautify